### PR TITLE
most of the newest library card design

### PR DESCRIPTION
**Pull request by 2is10**
Not updating the related library card yet because this design isn’t necessarily finalized. Need to let Danny & Mark play with it.
@yipingliao 

TODO: Add follow button.
TODO: Insert ellipsis at end of 3+ line titles.
TODO: Insert line breaks sometimes for more balanced title wrapping.

### CODE REVIEW

**Comment by 2is10** [comment]
Going to merge and deploy now so Danny can get it at 5am.

File: kifi-backend/modules/shoebox/angular/src/userProfile/userProfileLibrariesList.tpl.html [hunk 1]
```diff
@@ -6,22 +6,26 @@
...
     <div class="kf-upl-counts">
-      <a href="{{lib.path}}" class="kf-upl-count-link" ng-click="trackUplCardClick(lib)">
-        <ng-pluralize class="kf-upl-count" count="lib.numKeeps" when="{'1':'{} Keep','other':'{} Keeps'}"></ng-pluralize>
-      </a><a
-      href="javascript:" ng-if="lib.numFollowers > 0" class="kf-upl-count-link" ng-click="openFollowersList(lib)">
-        <ng-pluralize class="kf-upl-count" count="lib.numFollowers" when="{'1':'{} Follower','other':'{} Followers'}"></ng-pluralize>
+      <a href="{{lib.path}}" class="kf-upl-count" ng-click="trackUplCardClick(lib)">
+        <ng-pluralize count="lib.numKeeps" when="{'1':'{} Keep','other':'{} Keeps'}"></ng-pluralize>
+      </a>
+      <span class="kf-upl-count-dot" ng-if="lib.numFollowers > 0">·</span>
```
Comment (yipingliao) [inline kifi-backend/modules/shoebox/angular/src/userProfile/userProfileLibrariesList.tpl.html:19]: Thanks!

**Comment by yipingliao** [comment]
The new card looks great! I noticed that on the "libraries" tab, the curator's pic shows up on all the cards. Intentional?

**Comment by 2is10** [comment]
> I noticed that on the "libraries" tab, the curator's pic shows up on all
> the cards. Intentional
> 
> Intentional in the “it’s 11:30pm and this isn’t done yet” kind of way, yes.
> Fixing today.

Jared

**Pull request merged**